app.rb

```diff
@@ -1,8 +1,14 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
-    'Test page'
+    "Testing testing"
+  end
```

Comment:
It's always a good idea to remove test code from code you submit for code review (anywhere, not just at Makers)